wiki/osu!_rank_one_history/en.md

```diff
@@ -0,0 +1,265 @@
+- [ppv2](/wiki/Performance_points) (Since 27 January 2014)
+
+Note: During the period where Ranked Score was used, Rank 1 would often change as of such this list only includes users who have been verified to have reached Rank 1. Early dates use months where exact dates aren't available.
+
+## 2007
+
+| Player | Date |
+| --: | :-- |
+| ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) | 2007-10-4 |
+| ![][flag_AU] [eyup](https://osu.ppy.sh/users/88) | October 2007 |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-10-11 |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2007-12-24 |
+
+## 2008
+
+| Player | Date |
+| --: | :-- |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2008-10-07 |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | 2008-12-11 |
+
+## 2009
+
+| Player | Date |
+| --: | :-- |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-01-19 |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | January 2009 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | March 2009 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | May 2009 |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | June 2009 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | July 2009 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | July 2009 |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | July 2009 |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | September 2009 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | November 2009 |
+
+## 2010
+
+| Player | Date |
+| --: | :-- |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2010-02-07 |
+| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | 2010-09-08 |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | September 2010 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | October 2010 |
+| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | November 2010 |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | November 2010 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | December 2010 |
+
+## 2011
+
+| Player | Date |
+| --: | :-- |
+| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | January 2011 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | January 2011 |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | January 2011 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | Febuary 2011 |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | 2011-04-23 |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2011-05-31 |
+| ![][flag_TW] [Uan](https://osu.ppy.sh/users/147623) | 2011-06-05 |
+| ![][flag_KR] [chocomint](https://osu.ppy.sh/users/124493) | 2011-06-16 |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2011-11-05 |
+
+## 2012
+
+*Note: ppv1 was introduced on 20th April 2012.*
```

Comment:
Not sure if this is the correct way to apply ppv1: According to the wiki ppv1 was launched in April 2012 (with it being announced and released on 17 April 2012), but it only became the unique ranking system in-place only on 24 July 2012. Up until that date, ranks in the client were still displayed according to score (although the ppv1 leaderboard was available on the site). Since the client still used score based ranking, I think it's wrong to use ppv1 from this date. This is because of occourences such as this: http://geckogates.pristonworld.com/osu/gz_num1/ (link from geckogates' profile)

In the link you could se geckogates getting rank 1 in 21 April 2012, beating WubWoofWolf, and then an incredible thing happened: both WubWoofWolf and geckogates had the exact same score, resulting in both of them being rank 1. As you can see from the messages geckogates recieved, it seems the community, the client, and the website were all definitvely still judging rank according to ranked score, and so I'd that it'd conclude be wrong to apply ppv1 here.

Reply:
Yes, I must have gotten confused between when it got the pp name and when it became the unique system. 
```suggestion
*Note: ppv1 fully replaced Ranked Score rankings on 24 July 2012.*
```